async.js

```diff
@@ -0,0 +1,13 @@
+exports.handleLaunch = function(req, res) {
+  //call to the DB with alexaID
+  User.find({alexaID: req.params.alexaID})
```

Comment:
<img width="511" alt="screen shot 2016-08-25 at 11 12 52 am" src="https://cloud.githubusercontent.com/assets/13895848/17980603/f15c33cc-6ab4-11e6-83ed-e30769774bc0.png">

This is what I have in my code?? Not sure again why and or how it got changed but good catch.